bundle/metadata/annotations.yaml

```diff
@@ -5,7 +5,7 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: backstage-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.36.0
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.33.0
```

Comment:
it seems you're using an older version of the SDK (1.33) compared to the version used in the commmit that followed this one.